test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8378,6 +8378,8 @@ def cleanup_jobs(self, extend=None, runas=None):
         rv = self.expect(JOB, {'job_state': 0}, count=True, op=SET)
         if not rv:
             return self.cleanup_jobs(extend=extend, runas=runas)
```

Comment:
we won't have scheduling back to true if we return from here.

Reply:
I guess it wouldn't matter as the same function is recursively getting called.

Reply:
Ah yes missed the recursion.